cypress/e2e/contactForm.cy.js

```diff
@@ -21,10 +21,14 @@ describe('Contact', () => {
 
   it('should provide the ability to send feedback', () => {
     homePage.clickOnLink('Contact');
-    contactForm.typeEmail(testData.email);
-    contactForm.typeName(testData.name);
-    contactForm.typeMessage(testData.message);
-    contactForm.clickOnSendMessageBtn();
+    contactForm.emailField
+      .type(testData.email, { force: true });
```

Comment:
The main purpose of the POM approach is to hide all logic in the POM class. Store all your interactions (typing, assertions, clicks) within the POM methods.

Your previous ```contactForm.typeEmail(testData.email)``` is exactly what we need: in the test file we are calling the method, and all logic - interaction with a field - is hidden within the method ```typeEmail``` in the ```contactForm ``` class file